src/SUFST/inc/can_unpack.h

```diff
@@ -3,9 +3,25 @@
 #include "tx_api.h"
 #include "telemetry_protocol.h"
 
-extern TX_QUEUE queue_spi;
 
-UINT queue_receive_thread_create(TX_BYTE_POOL* stack_pool_ptr);
+#define TX_QUEUE_SIZE 10
```

Comment:
Again, I would prefix these constants with the name of the module. It also helps avoid re-definition of macros which can lead to all sorts of trouble. For example, if you included `can_publisher.h` in this file right now, the same `TX_QUEUE_SIZE` macro would be redefined.